SerratusApi/Controllers/RunsController.cs

```diff
@@ -28,55 +28,34 @@ public async Task<ActionResult<IEnumerable<Run>>> GetRuns()
             return await _context.Runs.ToListAsync();
         }
         
-        [HttpGet("get-run/{sra}")]
-        public async Task<ActionResult<Run>> GetSummary(string sra)
+        [HttpGet("get-runs/{run}")]
+        public async Task<ActionResult<Run>> GetSummaryForSraAccession(string run)
         {
 
-            var run = await _context.Runs.FirstOrDefaultAsync(r => r.Sra == sra);
+            var sraAccessions = await _context.Runs.FirstOrDefaultAsync(r => r.Sra == run);
```

Comment:
`sraAccession` would be a better name (singular)

Reply:
Agreed